keyboards/themadnoodle/noodlepad_micro/config.h

```diff
@@ -4,3 +4,7 @@
 #pragma once
 
 #define RGBLIGHT_DEFAULT_MODE RGBLIGHT_MODE_RAINBOW_SWIRL + 5
+#define RGBLIGHT_LAYERS
+#define RGBLIGHT_LAYER_BLINK
+#define RGBLIGHT_MAX_LAYERS 8
+#define RGBLIGHT_LAYERS_OVERRIDE_RGB_OFF
```

Comment:
These should be moved to info.json.  `qmk info -f json` can help with this.

Reply:
I had tried to add these to the JSON but they didn't work that's why they are in the config. So wasn't sure if they were implemented into to the JSON yet. 
If they are. Can you show me how to add them? 
All the ways I tried failed. 

If you don't mind be asking what does the `qmk info -f json` command do? I tried to use both `qmk info -f json -kb themadnoodle/noodlepad_micro`  and `qmk info -f json -kb themadnoodle/noodlepad_micro -km via` but nothing changed.

Reply:
This what i get when i try to add those to the JSON

```
☒ Not including data from file: keyboards/themadnoodle/noodlepad_micro/info.json
☒       rgblight: Additional properties are not allowed ('layer_blink', 'layers_override_rgb', 'max_layers' were unexpected)
☒ 'matrix_size'
Traceback (most recent call last):
  File "C:/QMK_MSYS/mingw64/lib/python3.10/site-packages/milc/milc.py", line 528, in __call__
    return self.__call__()
  File "C:/QMK_MSYS/mingw64/lib/python3.10/site-packages/milc/milc.py", line 533, in __call__
    return self._subcommand(self)
  File "C:/madnoodlefirmware/qmk_firmware/lib/python/qmk/cli/generate/rules_mk.py", line 65, in generate_rules_mk
    kb_info_json = dotty(info_json(cli.args.keyboard))
  File "C:/madnoodlefirmware/qmk_firmware/lib/python/qmk/info.py", line 221, in info_json
    info_data = _extract_led_config(info_data, str(keyboard))
  File "C:/madnoodlefirmware/qmk_firmware/lib/python/qmk/info.py", line 717, in _extract_led_config
    cols = info_data['matrix_size']['cols']
KeyError: 'matrix_size'
```